docs/static/breaking-changes.asciidoc

```diff
@@ -45,9 +47,30 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
  The new mapping template has been updated to reflect that. If you are using a custom mapping template you may need to update it to reflect that.
 
 [float]
+==== Kafka input changes
+
+* Updated to 0.10.1.0 client protocol. Only supports brokers 0.10.1.x or later
+* Decorated fields are now nested under `@metadata` to avoid mapping conflicts with beats
+* The `ssl` option is now obsolete
+
+[float]
+==== Kafka output changes
+
+* Updated to 0.10.1.0 client protocol. Only supports brokers 0.10.1.x or later
+* The options `block_on_buffer_full`, `ssl` and `timeout_ms` are now obsolete
+
+[float]
+==== Beats input changes
+
+* Logstash will no longer start when multiline codec is used with the Beats input plugin
```

Comment:
Can we suggest corrective action for this?

This line says what is not working, but for users affected, giving a pointer to the next step is useful --

"It is recommended to use filebeat's multiline support as a replacement." or some similar language?

Reply:
Could we also point to the FB's multiline documentation? 

https://github.com/logstash-plugins/logstash-input-beats/issues/201#issuecomment-304729091